src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,207 @@
+            throw new DataProcessingException("Can't create car " + car, e);
+        }
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String getRequest = "SELECT cars.id, model, manufacturer_id, "
+                + "manufacturers.name AS mark, country FROM cars "
+                + "JOIN manufacturers ON cars.manufacturer_id = "
+                + "manufacturers.id WHERE cars.is_deleted = FALSE AND cars.id = ?;";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getStatement =
+                             connection.prepareStatement(getRequest)) {
+            getStatement.setLong(1, id);
+            ResultSet resultSet = getStatement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCarWithDriverFromResultSet(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't find book in DB with id= " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getDriversForCar(id));
+        }
+        return Optional.ofNullable(car);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String getAllRequest = "SELECT cars.id, model, manufacturer_id, "
+                + "manufacturers.name AS mark, country FROM cars "
+                + "JOIN manufacturers ON cars.manufacturer_id = "
+                + "manufacturers.id WHERE cars.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getAllStatement =
+                         connection.prepareStatement(getAllRequest)) {
+            ResultSet resultSet = getAllStatement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(parseCarWithDriverFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't read all cars from DB", e);
+        }
+        cars.forEach(i -> i.setDrivers(getDriversForCar(i.getId())));
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String updateRequest = "UPDATE cars SET model = ?, manufacturer_id = ? WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateStatement =
+                         connection.prepareStatement(updateRequest)) {
+            updateStatement.setString(1, car.getModel());
+            updateStatement.setLong(2, car.getManufacturer().getId());
+            updateStatement.setLong(3, car.getId());
+            updateStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update a car " + car, e);
+        }
+        removeAllRelations(car.getId());
+        insertDrivers(car);
```

Comment:
let's use consistent naming. better rename `removeAllRelations` to `removeDrivers`